ansible_collection/hpe/nimble/plugins/doc_fragments/hpe_nimble.py

```diff
@@ -38,9 +38,9 @@ class ModuleDocFragment(object):
     type: str
 requirements:
   - Ansible 2.9 or later
-  - Nimble OS 5.0 or later
+  - NimbleOS 5.0 or later
   - HPE Nimble Storage SDK for Python 1.0.0 or later (nimble-sdk Python module)
```

Comment:
we may need to request `Python 1.1.0` ?

Reply:
i don't think so.. our ansible modules can also work with 1.0 as we do not make a call to that api.(forgot the name :( . for which we made the change.. so we are good